genesis/src/lib.rs

```diff
@@ -319,7 +334,12 @@ mod tests {
         };
         let _genesis_block = GenesisNetwork::from_configuration(
             true,
-            RawGenesisBlock::new("alice".parse()?, "wonderland".parse()?, alice_public_key),
+            RawGenesisBlockBuilder::new()
+                .domain("wonderland".parse()?)
+                .account("alice".parse()?, alice_public_key)
+                .finish_domain()
+                .validator(MockValidator)
```

Comment:
considering that none of these tests ever use this validator, you can do this:

```suggestion
                .validator(Validator::new(WasmSmartContract::new(Vec::new()))
```

I don't know if this is something you can do in `iroha_core`